hack/build/ci/snyk-image-import.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -x
+
+readonly snyk_org_id=${1}
+readonly snyk_int_id=${2}
+readonly snyk_api_token=${3}
+readonly image=${4}
+
+response=$(curl --include \
+     --request POST \
+     --header "Content-Type: application/json; charset=utf-8" \
+     --header "Authorization: token ${snyk_api_token}" \
+     --data-binary "{
```

Comment:
can we split the payload into a separate variable and build it later so it's easier to read?